cmd/litcli/autopilot.go

```diff
@@ -20,14 +20,20 @@ var autopilotCommands = cli.Command{
 		{
 			Name:      "features",
 			ShortName: "f",
-			Usage:     "List available features",
-			Action:    listFeatures,
+			Usage:     "List available Autopilot features.",
+			Description: `
+			List available Autopilot features.	
```

Comment:
Hmm, this creates an indentation in the generated text of 3 levels. I know it looks weird in the code here, but I think we only want one indentation as we have in the accounts commands (and lncli in general).

Reply:
ahhhh good catch! took me a while to figure out what you mean. I have moved each command into its own named `var` so that it doesnt need to look weird :)